### PR TITLE
dependency-check 2.0.0

### PR DESCRIPTION
**Pull request by colezlaw**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ilovezfs** [comment]
@colezlaw https://www.owasp.org/index.php/OWASP_Dependency_Check still shows 1.4.5 as the current release.

**Comment by colezlaw** [comment]
@ilovezfs that's been fixed. I guess we have a (somewhat) circular dependency there. We tend to update the version number on the website once all the available mechanisms are updated.

**Comment by ilovezfs** [comment]
@colezlaw no problem. Thanks for clarifying.

File: Formula/dependency-check.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def install
     output = shell_output("#{libexec}/bin/dependency-check --version").strip
     assert_match "Dependency-Check Core version #{version}", output
 
-    (testpath/"temp-props.properties").write "cve.startyear=2017"
+    (testpath/"temp-props.properties").write "cve.startyear=2017\nanalyzer.assembly.enabled=false"
```
Comment (ilovezfs) [inline Formula/dependency-check.rb]: It would be preferable to use a heredoc.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

**Pull request closed without merging**